src/apps/external_executor/executor_code_id.h

```diff
@@ -16,6 +16,12 @@ struct ExecutorNodeInfo
   ccf::Attestation attestation;
   std::vector<ccf::NewExecutor::EndpointKey> supported_endpoints;
 };
+using ExecutorId = ccf::EntityId<ccf::NodeIdFormatter>;
+typedef std::map<ExecutorId, ExecutorNodeInfo> ExecutorIDMap;
+typedef std::map<ExecutorId, crypto::Pem> ExecutorCertsMap;
+
+ExecutorIDMap ExecutorIDs;
+ExecutorCertsMap ExecutorCerts;
```

Comment:
```suggestion
ExecutorIDMap executor_ids;
ExecutorCertsMap executor_certs;
```

Not auto-enforced, but our coding style is `UpperCamelCaseTypes` and `lower_snake_case_variables`.